doc/howto/usage/k8s/k8s_aws_en.md

```diff
@@ -1,7 +1,14 @@
+
 # Distributed PaddlePaddle Training on AWS with Kubernetes
 
 We will show you step by step on how to run distributed PaddlePaddle training on AWS cluster with Kubernetes. Let's start from core concepts.
 
+## Choose AWS Service Region
```

Comment:
Maybe it's better to put this into Aws section, specifically, before `Configure AWS Credentials`, since that's the section that user first time select a region.